flashlight/memory/managers/CachingMemoryManager.cpp

```diff
@@ -88,6 +96,25 @@ CachingMemoryManager::CachingMemoryManager(
 
 void CachingMemoryManager::initialize() {}
 
+void safeStoull(size_t& var, const char* str) {
```

Comment:
Let's add this function in an anonymous namespace at the top of this file

Reply:
done